src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -27,6 +25,13 @@ export default function NewCapabilityDialog({
 
   const [formData, setFormData] = useState(emptyValues);
   const [invitees, setInvitees] = useState([]);
+  /*
+  const [tagFormData, setTagFormData] = useState({});
+
+  useEffect(() => {
+    console.log(tagFormData);
+  }, [tagFormData]);
+  */
```

Comment:
Maybe rewrite the code to only show the tags if there are any required fields? that way we don't need another PR for enabling the code

Reply:
This is purely debug and should have been removed; I have torn it out in the continuation project, so this will be gone soon.